nautobot/docs/installation/migrating-from-netbox.md

```diff
@@ -16,7 +16,7 @@ Although Nautobot will run perfectly well with a default configuration (such as
 
 Due to a number of significant infrastructural changes between the applications, you cannot simply point Nautobot at your existing NetBox PostgreSQL database and have it automatically load your data. Fortunately, Network to Code (NTC) and collaborators have developed a Nautobot plugin, `nautobot-netbox-importer`, that can be used to import a NetBox database dump file into Nautobot. For full details, refer to [the plugin's own documentation](https://github.com/nautobot/nautobot-plugin-netbox-importer/), but in brief, the following steps will be required:
 
-1. Export your NetBox database to a JSON file using the `manage.py dumpdata` command
+1. Export your NetBox database to a JSON file using a [manage.py command](https://github.com/nautobot/nautobot-plugin-netbox-importer#getting-a-data-export-from-netbox)
```

Comment:
I see why you're suggesting this, since some people might think to just run `manage.py dumpdata` without any arguments from their NetBox instance.

Since we're already duplicating procedural steps for `nautobot-netbox-importer` here anyways, I feel like should we should just also duplicate the exact command that needs to be executed, but still have the link. Thoughts?

Reply:
I'd rather just get rid of the "in brief" listing here if it's causing confusion. Otherwise we'll have to update these docs if/when the importer procedure changes in any detail.

Reply:
It's the classic DRY vs usable :) Would gather feedback from @glennmatthews as well.

Reply:
If @glennmatthews is happy I'm happy. 🙃